lib/esm-utils.js

```diff
@@ -15,7 +12,7 @@ const requireOrImport = async file => {
     return require(file);
   } catch (err) {
     if (err.code === 'ERR_REQUIRE_ESM') {
-      return import(url.pathToFileURL(file));
```

Comment:
Not sure what `url.pathToFileURL` was achieving?
`import()` allows a plain string, and this way the required file can still be resolvable via node's logic (e.g. loading from `node_modules`)

Reply:
I guess it was required for Windows.
Gosh I hate windows